### DISCUSSION TITLE
Making Real Dev Squad apps better

### DISCUSSION THREAD

**Discussion by ankushdharkar** [Real-Dev-Squad/website-todo-items#97] [Ideas]
What can be done to make our products better? What are our pain-points?

**Comment by RitikJaiswal75** [comment]
Signout on pages other than website-www

**Comment by pallabez** [comment]
We need to improve our design and branding. Hiring a designer would be a good idea and to make sure their designs don't stagnate.

**Comment by RitikJaiswal75** [comment]
Login issue on status site

**Reply by ankushdharkar** [reply]
@Maheima is working on this

**Comment by pallabez** [comment]
Improve first site load time. Paid heroku or switching to some other paas. 

Also adding a caching layer in backend for improved API performance.

**Comment by alishvagadiya** [comment]
Initial mock data for backend setup.
After validate-setup, 
two yarn command
- prefill-data (for adding mock data in local/personal DB)
- clean data (which will clean data from DB)
